cli/running/running.go

```diff
@@ -155,6 +155,15 @@ loop:
 	return result
 }
 
+// createDataDir checks if datadir folder exists, if not makes it.
```

Comment:
`if datadir folder` - "if `DataDir` directory"
`makes it` -> `creates it`? (`createDataDir`)